src/App.js

```diff
@@ -125,44 +126,52 @@ const App = () => {
   }
 
   return (
-    <div>
-      <UserForm onRegister={handleRegister} onLogin={handleLogin} onLogout={handleLogout}/>
-
-      <br/>
-      <br/>
-
-      <CollectionNameInput title='Get collection: ' onSubmit={handleGetCollection}/>
-      <CollectionItemInput title='Get collection item: ' onSubmit={handleGetItem}/>
-      <CollectionItemInput title='Create collection item: ' onSubmit={handleCreateItem} isJson={true}/>
-      <CollectionItemInput title='Update collection item (PATCH): ' onSubmit={handleOverwriteItem} isUpdate={true}/>
-      <CollectionItemInput title='Update collection item (PUT): ' onSubmit={handleUpdateItem} isUpdate={true}/>
-      <CollectionItemInput title='Delete collection item: ' onSubmit={handleDeleteItem}/>
-
-      <br/>
-      <br/>
-
-      <RunCcf onSubmit={handleRunCcf}/>
-
-      <br/>
-      <br/>
-
-      <FileInput title='Get all files: ' onSubmit={handleGetAllFiles}/>
-      <FileInput title='Get file: ' onSubmit={handleGetFile}/>
-      <FileInput title='Delete file: ' onSubmit={handleDeleteFile}/>
-
-      <br/>
-      <br/>
-
-      <FileForm onSend={handleSendFile}/>
-
-      <br/>
-      <br/>
-
-      <div>
-        Your result:
-        <p className="result">
-          {JSON.stringify(resultSection)}
-        </p>
+    <div className='h-screen'>
+      <Header />
+      <div className='flex flex-col items-center h-full' id='main'>
+        {/* PAGE /USERS */}
+        <div className="flex-auto flex flex-col items-center max-w-screen-lg pt-12 pb-2 mb-64 h-full">
+          <p className="text-2xl text-black mb-2">User registration</p>
+          <UserForm onRegister={handleRegister} onLogin={handleLogin} onLogout={handleLogout}/>
+        </div>
+        {/* END OF PAGE /USERS */}
+
+        {/* PAGE /COLLECTIONS */}
+        <div className="flex-auto flex flex-col items-center max-w-screen-lg pt-12 pb-2 mb-64 h-full">
+          <p className="text-2xl text-black mb-2">Data operations</p>
+        {/* CAN WE PUT JSON DISPLAY THING AT THE TOP HERE TO SHOW RESULTANT DATA? */}
```

Comment:
We need to find something to display data in for our demo.
If it is not too hard to set up, can we have the structured JSON panel you created for the `admin-frontend` @partemen86 ?
This one
![image](https://user-images.githubusercontent.com/14999320/160458963-68e157c8-969e-4df5-a87b-7a22dba906aa.png)